.github/workflows/env-publish-test-base-image.yaml

```diff
@@ -0,0 +1,27 @@
+name: (k8s package) Publish Test Base Image
+on:
+  push:
+    tags:
+      - 'v*'
+    paths:
```

Comment:
This won't work as expected since new tags/releases can have more than 1 commit associated with them. The way you currently have the logic this would only work if the k8s changes happened to happen in the commit that the tag is based on. If the k8s changes happened in the commit just before it then it would not create a new release.

Reply:
do you think we need some kind of more sophisticated filter here or we should just run it when when a tag is added?